embedded-graphics/src/primitives/rectangle.rs

```diff
@@ -97,15 +97,31 @@ impl Rectangle {
         }
     }
 
+    fn with_center(center: Point, size: Size) -> Self {
```

Comment:
Should we make this `pub`?

Reply:
Yeah I think that's a good idea. Might be worth mentioning the even height behaviour like the `fn center` doc comment.

Reply:
Done. I hope the comment is OK.

Reply:
Yep! Looks good